packages/teams-js/src/internal/chatUtilities.ts

```diff
@@ -0,0 +1,20 @@
+import {
```

Comment:
Looking forward to the work Jay will have to do for the non-chat capabilities that are like this, would it make more sense to name this file something like deepLinkUtilities? That way similar functionality could be put in here

Reply:
My take on things like this is that it's good to think about future extensibility but given that the main thing we would do to potentially support that in the future is just renaming, I'd prefer to wait until we have the second consumer of this and then rename it, rather than pre-emptively rename it and then somehow discover later that Jay's work has to be done in a slightly different way. Said differently: I hope this is helpful to Jay, and it should be very easy for him to rename it when he's ready.